packages/blocks/src/CookieBanner/CookieBanner.tsx

```diff
@@ -89,6 +89,7 @@ export const CookieBanner: FC<CookieBannerProps> & CookieBannerStaticMembers = (
 
 const CookieBannerContent: FC<AlertProps> = ({ children, ...props }) => (
   <Alert variant="subtle" pt="0" pr={{ base: '0', lg: '2rem' }} pb={{ base: '2rem', lg: '0' }} {...props}>
+    <Alert.Icon />
```

Comment:
I have updated this file to avoid chromatic from complaining, but I am not sure if we need to create a new version first of `components` package and then update `blocks` with this new version and update `CookieBanner`